pkg/antctl/transform/networkpolicy/response_test.go

```diff
@@ -176,17 +177,18 @@ func TestListTransform(t *testing.T) {
 
 func TestEvaluationResponseTransform(t *testing.T) {
 	test := EvaluationResponse{&cpv1beta.NetworkPolicyEvaluation{}}
-	assert.Equal(t, []string{"NAME", "NAMESPACE", "POLICY-TYPE", "RULE-INDEX", "DIRECTION"}, test.GetTableHeader())
+	assert.Equal(t, []string{"NAME", "NAMESPACE", "POLICY-TYPE", "RULE-INDEX", "DIRECTION", "ACTION"}, test.GetTableHeader())
 	assert.False(t, test.SortRows())
-	assert.Equal(t, []string{"", "", "", "", ""}, test.GetTableRow(32))
+	assert.Equal(t, []string{"", "", "", "", "", ""}, test.GetTableRow(32))
+	testAction := crdv1beta1.RuleActionAllow
 	test.Response = &cpv1beta.NetworkPolicyEvaluationResponse{
 		NetworkPolicy: cpv1beta.NetworkPolicyReference{
 			Type:      cpv1beta.K8sNetworkPolicy,
 			Namespace: "ns",
 			Name:      "testName",
 		},
 		RuleIndex: 10,
-		Rule:      cpv1beta.RuleRef{Direction: cpv1beta.DirectionIn},
+		Rule:      cpv1beta.RuleRef{Direction: cpv1beta.DirectionIn, Action: &testAction},
 	}
-	assert.Equal(t, []string{"testName", "ns", "K8sNetworkPolicy", "10", "In"}, test.GetTableRow(32))
+	assert.Equal(t, []string{"testName", "ns", "K8sNetworkPolicy", "10", "In", "Allow"}, test.GetTableRow(32))
```

Comment:
Quite misleading UT. K8s NP should not have action. We should use a testcase to test action "" and another at least to test outputting a real action

Reply:
That makes sense, but this also reminds me that for the manually inserted K8s default isolation rules, it's better to assign "Drop" to them for identification?